lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -159,6 +159,10 @@ class ReportDownloads
   constructor: (@$section) ->
 
     @$report_downloads_table = @$section.find ".report-downloads-table"
+    @$reports                         = @$section.find '.reports-download-container'
```

Comment:
Is `this.reports` actually being used elsewhere or is it just used as a local variable?